### PR TITLE
Fall-back to image name as key when there is no id

### PR DESCRIPTION
**Pull request by koddsson**
When a container is first started and the image is downloading there is no id set. With this patch we'll check if the container has a id and and is downloading, if so we'll substitute it with the image name and render such.

This will fix the react key props warning in the developer console as well as maybe give us some sort of performance boost(?)

### CODE REVIEW

File: src/ContainerList.react.js [hunk 1]
```diff
@@ -9,8 +9,14 @@ var ContainerList = React.createClass({
   render: function () {
     var self = this;
     var containers = this.props.containers.map(function (container) {
+      var container_id = container.Id;
```
Comment (jeffdm) [inline src/ContainerList.react.js]: containerId (camel case :)

**Comment by jeffdm** [comment]
Thanks for the PR! Added a small comment.

File: src/ContainerList.react.js (hunk 1 above)
Comment (koddsson) [inline src/ContainerList.react.js]: Sounds good! :)

Comment (koddsson) [inline src/ContainerList.react.js]: Also would you consider something like [jscs](http://jscs.info/) to automatically check those rules? (I've already added `requireCamelCaseOrUpperCaseIdentifiers` to my local `.jscsrc` ;)

**Comment by jeffdm** [comment]
Any way to sign that last commit? Sorry about that.

Working on an easy way to sign once for all commits or similar.

**Comment by koddsson** [comment]
I've forced pushed and added a pre-commit hook to remind me! :)

**Comment by jeffdm** [comment]
Great! Sorry for the delay.

LGTM! Merging

**Pull request merged**